data/blastr.md

```diff
@@ -0,0 +1,22 @@
+---
+title: blastr
+web: 
+git: https://github.com/MutinyWallet/blastr
+instances:
+tags:
+- tool
+platforms:
+- desktop
```

Comment:
No new one-off platforms, please. This would be Linux, Windows and Mac then?

Reply:
missed this, sorry. Yes, Linux, Windows and Mac as it can be installed via npm: 

> With wrangler, you can build, test, and deploy your Worker with the following commands:

# install wrangler if you do not have it yet
$ npm install -g wrangler

# log into cloudflare if you havent before
$ wrangler login

# compiles your project to WebAssembly and will warn of any issues
$ npm run build

# run your Worker in an ideal development workflow (with a local server, file watcher & more)
$ npm run dev

# deploy your Worker globally to the Cloudflare network (update your wrangler.toml file for configuration)
$ npm run deploy